person.rb

```diff
@@ -20,6 +22,10 @@ def correct_name
     @name
   end
 
+  def add_rental(date, person)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ] Rental can be created with ,date, person, & book parameter.Kindly change according to the requirement.
```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```